lib/livebook_web/live/session_live.ex

```diff
@@ -636,19 +663,22 @@ defmodule LivebookWeb.SessionLive do
   end
 
   def handle_event("delete_cell", %{"cell_id" => cell_id}, socket) do
+    assert_policy!(socket, :read)
```

Comment:
@jonatanklosko should this be edit?

Reply:
Definitely! Pretty sure it changed since when I checked ^^